tests/sys_arduino_analog/arduino-test.sketch

```diff
@@ -0,0 +1,15 @@
+// Example sketch given in analogWrite documentation
+// https://www.arduino.cc/reference/en/language/functions/analog-io/analogwrite/
+
+int ledPin = 3;      // LED connected to digital pin 3
```

Comment:
Since all boards use a different pin mapping, it could be better to use
```suggestion
#ifdef ARDUINO_LED
int ledPin = ARDUINO_LED;   // built in LED is used
#else
int ledPin = 3;             // LED connected to digital pin 3
#endif
```
which corresponds to the most basic arduino example `blink`. Here I'm wondering why our Arduino compatibility layer defines `ARDUINO_LED` instead of `BUILTIN_LED`.

Reply:
It depends, I think not all boards define the ARDUINO_LED with PWM capability.

Reply:
I checked and Arduino Zero doesn't have a PWM configuration for the on board LED.

Reply:
Ah ok, I see. Would be a define that can be changed by `CFLAGS` at make command line help?

Reply:
I added a variable to be able to pass the pin number via CFLAGS.